Undocument `command-nightly` model alias (#582)
cohere-ai/cohere-developer-experience: fern/pages/llm-university/sandbox/creating-custom-generative-models-copy.mdx

<img src='../../../assets/images/09f2b0e-image.png' alt='task' />

> At Cohere, we refer to our pre-trained models as baseline models (at the time of writing, these are `command-light-nightly`, `base`, and `base-light`). We refer to finetuned models as custom models.

Author: ```suggestion
At Cohere, we refer to our pre-trained models as baseline models and we refer to fine-tuned models as custom models.
```
